### PR TITLE
[next] feat(NcUserBubble): migrate to vue 3

### PR DESCRIPTION
**Pull request by raimund-schluessler**
### ☑️ Resolves

* This migrates `NcUserBubble` to vue 3.

### CODE REVIEW

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
Works but noticed an error (we also have this on master so unrelated^^)

File: src/components/NcUserBubble/NcUserBubble.vue [hunk 1]
```diff
@@ -91,7 +101,7 @@ This component has the following slot:
 					:style="styles.avatar"
 					:disable-tooltip="true"
 					:disable-menu="true"
-					v-bind="$props"
+					v-bind="$attrs"
```
Comment (ShGKme) [inline src/components/NcUserBubble/NcUserBubble.vue:104]: I haven't tested this specific place, but this looks like a behavior change. Is it migration-related?

For example, prop `showUserStatus` prop is unused now.

Probably, we need to fix this on `main` first

Comment (raimund-schluessler) [reply src/components/NcUserBubble/NcUserBubble.vue]: Indeed, that's a bit inconsistent. I changed it now to explicitly only forward the props that `NcAvatar` uses. If this is ok, I would send a PR to `master` as well.

Comment (raimund-schluessler) [reply src/components/NcUserBubble/NcUserBubble.vue]: We should anyway wait until https://github.com/nextcloud-libraries/nextcloud-vue/pull/4743 is merged and backported before continuing here.

Comment (raimund-schluessler) [reply src/components/NcUserBubble/NcUserBubble.vue]: See #4756 for the fix for `master`.

Comment (raimund-schluessler) [reply src/components/NcUserBubble/NcUserBubble.vue]: With #4756 merged into `next`, the problem should be solved.

**Review by ShGKme — APPROVED** [review]

**Pull request merged**